docs/users-guide/models.md

```diff
@@ -0,0 +1,107 @@
+# Models
+
+Models are a fundamental building block in Metabase. Models curate data from another table or tables from the same database to anticipate the kinds of questions people will ask of the data. You can think of them as derived tables, or a special kind of saved question meant to be used as the starting point for new questions.
+
+Models:
+
+- Let you update column descriptions and customize metadata to create great starting points for exploration.
+- Show up higher in search results and get highlighted when other users start new questions to promote reuse.
+- Live in collections to keep them separate from messy database schemas.
+
+## How to use models
+
+You can use models to:
+
+- Create, uh, models, with model here meaning a simplified description of some concept in your business that you codify as a set of columns. An example model could be a "customer", which is a table that pulls together customer information from multiple tables and adds computed columns, like adding a lifetime value (LTV) column. This model represents the [measures and dimensions][measures-dimensions] that you think are relevant to your understanding of your customers.
```

Comment:
> a simplified description

Maybe more like an "intuitive encapsulation"? "Simplified" almost carries the negative connotation that it's less robust or accurate.